include/tue/config/reader_writer.h

```diff
@@ -225,7 +226,7 @@ class ReaderWriter
      * @param filename input file
      * @return indicates success
      */
-    bool loadFromYAMLFile(const std::string& filename);
+    bool loadFromYAMLFile(const std::string& filename, const ResolveConfig& resolve_config=ResolveConfig());
```

Comment:
For all these default arguments it might be preferable to define 'defaultconfig' rather than rely on the ResolveConfigs default arguments.

Reply:
Done